libavfilter/vf_vpp_qsv.c

```diff
@@ -557,10 +557,8 @@ static int activate(AVFilterContext *ctx)
             ret = ff_qsvvpp_filter_frame(qsv, inlink, in);
             av_frame_free(&in);
 
-            if (s->eof) {
-                ff_outlink_set_status(outlink, status, pts);
-                return 0;
-            }
+            if (s->eof)
+                goto eof;
```

Comment:
I wonder whether returning 0 from this callback makes sense when ff_qsvvpp_filter_frame() returns an error  and s->eof is true

Reply:
Yes，I also notice here may leak of checking and return if ret < 0. I will try to see if can add the fix into this patch, otherwise, I will submit another one.

Reply:
If returns error, the cmd will be assert directly. So if there is error, return it before set eof should be fine.